### DISCUSSION TITLE
server/api

### DISCUSSION THREAD

**Discussion by darrychui** [nuxt/nuxt#20655] [Questions] [answered]
I watched @danielroe on vue.js forge using the server/api to create a server endpoint for the client to interact.  My question is, how can you make that api endpoint restricted to auth clients only.  thanks

**Comment by pi0** [comment, marked as answer]
Hi. Please check this example: 

https://github.com/nuxt/examples/blob/main/auth/local/server/api/secret.ts

https://github.com/nuxt/examples/tree/main/auth/local

**Reply by darrychui** [reply]
Thanks
